### PR TITLE
Circleci trigger image build workflow

### PR DESCRIPTION
**Pull request by eguzki**
CircleCI job to build 3scale components images and push to quay.io image repo. Includes Makefile target to make it easy to trigger building job.

Depends on https://github.com/3scale/3scale-operator/pull/554. Needs to be merged after.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -132,7 +132,7 @@ controller-gen:
 ifeq (, $(shell which controller-gen))
 	@{ \
 	set -e ;\
-	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
+	CONTRjLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
```
Comment (miguelsorianod) [inline Makefile:135]: Introduced change with a typo

**Review by miguelsorianod — CHANGES_REQUESTED** [review]
Looks good.

Fix the error in the comment I left and rebase and all ok :+1:

File: Makefile (hunk 1 above)
Comment (eguzki) [reply Makefile]: good catch!

**Comment by eguzki** [comment]
Ready for review @miguelsorianod

**Review by miguelsorianod — APPROVED** [review]
:+1:

**Pull request merged**